docs/melange-book/melange-playground.md

```diff
@@ -0,0 +1,97 @@
+# Melange Playground
+
+[Melange Playground](https://melange.re/unstable/playground/) is an interactive
+environment for running code and seeing its output, including the JavaScript
+generated by Melange compiler. It's also a great way to share code
+snippets--the URL can be copied and pasted into another tab, recreating the
+exact text inside the code editor.
+
+In the next section, we'll use the Melange Playground to explore OCaml's numeric
+types.
+
+## OCaml numeric types
+
+Unlike JavaScript, OCaml makes a clear and hard distinction between integer and
+float values. Try running the following program in the playground:
+
+```reason
+let foo = 42;   // int
+let bar = 42.0; // float
+Js.log(foo == bar);
+```
+
+You'll get a compiler error for the last line that says `This expression has
+type float but an expression was expected of type int`. Unlike JavaScript, there
+are no implicit conversions in OCaml. Therefore you cannot expect to compare an
+integer and a float together, unless you convert one of the values so that both
+values have the same type. To make the last line compile, you can change it to
+
+```reason
+Js.log(foo == Int.of_float(bar));
+```
+
+Another way to fix the last line is to convert `foo` from an int to a float:
+
+```reason
+Js.log(Float.of_int(foo) == bar);
+```
+
+What about addition? What from you've already seen, you can probably guess that
+`Js.log(42 + 16.0)` won't compile. However, you may be surprised to discover
+that `Js.log(42.0 + 16.0)` also won't compile! That's because OCaml uses
+separate arithmetic operators for floats. What will compile is this:
+
+```reason
+Js.log(42.0 +. 16.0);
+```
+
+Underneath the covers, `foo` and `bar` are both instances of JavaScript's Number
+type:
+
+```reason
+Js.log(Js.typeof(foo)); // prints "number"
+Js.log(Js.typeof(bar)); // prints "number"
+```
+
+## Overview
```

Comment:
In another section, "Overview" was written between "Exercises" and "Solutions", but here it's before both of them.

Reply:
Fixed in https://github.com/melange-re/melange-re.github.io/pull/93/commits/25b13fb50d3bc84d10649da6f383b26e6d39486b